test/e2e-tests.sh

```diff
@@ -45,9 +45,6 @@ initialize $@
 
 header "Running tests"
 
-# TODO: Real integration tests here instead of just verifying that we
-# can list Knative Services.
-./kn service get || fail_test
-echo "kn service get returned success"
+go_test_e2e ./test/e2e
```

Comment:
Sorry, I should have given you the full command line.

```suggestion
go_test_e2e ./test/e2e || fail_test
```

Reply:
Sweet, now the failures are shown in Gubernator. :)